ca_gazebo/models/create_camera/model.sdf

```diff
@@ -10,23 +10,37 @@
           </box>
         </geometry>
       </visual>
-      <sensor name='my_camera' type='camera'>
+      <sensor name='apriltag_camera' type='camera'>
         <camera>
-          <save enabled="true">
+          <save enabled="false">
             <path>/tmp/camera_save_tutorial</path>
           </save>
           <horizontal_fov>1.047</horizontal_fov>
           <image>
-            <width>1920</width>
-            <height>1080</height>
+            <width>5760</width>
```

Comment:
Isn't better to increase the FoV or decrease the size of the environment?

Reply:
Definitely decrease the size. I forgot to do it. I focused initially on making work the detection.
Gonna address this.